script/Ciel/Day 5/05_01_CIEL05_3E.txt

```diff
@@ -740,43 +787,55 @@ As far as the executor is concerned, her true weapon of choice is the Mystic Cod
 [sha:25240e7102d864008042dae61ef55425b2ea8069]{
 -- Page 627, Offset 12501.
 -- 少女は自らに向けて詠唱する。
-The girl chants unto herself.
+The girl chants to herself.
 }
 [sha:bdcd145c5f7f1263009ebe4b74c81ce2549b2744]{
 -- Page 627, Offset 12502.
 -- 現状、銃器による制圧は不可能だと判断した。
-In her current state, she has come to the conclusion that she cannot overwhelm him with her firearm.
+Having evaluated the situation, she's come to the conclusion that she cannot overwhelm him with firearms.
 }
 [sha:97b7d06f9cdbfeaab433dd4f07879cb6d87a843e]{
 -- Page 627, Offset 12503.
 -- であれば、<刃|は>を散らす白兵戦の時間になる……！
-If so, then the time has come to release her blade and fight at close quarters...!
+In which case, the time has come to cross blades and engage in hand-to-hand combat...!
 }
 [sha:d88db2d0964c9504473c31a41a89344e84ba2555]{
 -- Page 628, Offset 12504.
 -- 牙を<剥|む>く聖女の鉄槌。
-The iron hammer of the holy woman bares its fangs.
+// 聖女 was previously used to refer to the ferrous female and capitalized, I guess is this part of the same accoutrements?
+The iron hammer of the Holy Maiden bares its fangs.
 }
 [sha:018116cfba72d8f2e8000f81bcc40b26f1518658]{
 -- Page 628, Offset 12505.
 -- 少女に与えられたものは銃と甲冑だけではない。
-It isn't only the armor and the gun that are entrusted to the girl.
+The armor and the gun aren't the only things the girl has been entrusted with.
 }
 [sha:32801955e57056687c574801d44ec84b6ed84ecb]{
 -- Page 628, Offset 12506.
 -- <焼死|ブレイズ>。<病死|シック>。<出血死|ブレイド>。<衝突死|ブレイク>。<精|ロ><神|ス><死|ト>。<拷|ペ><問|イ><死|ン>。<断罪死|パニッシュ>。
-// I tried to fancy up some of the terms. I also changed sickness because of the Sick as ruby, not great to have that repetition. 精神死 gives me some "have no more mind" vibes, rather than "insane".
-<Death through Immolation|B l a z e         >. <Death through Disease|S i c k            >. <Death through Hemorrhage|B l a d e          >. <Death through Concussion|B r e a k           >. <Death through Dementia|L o s t        >. <Death through Torture|P a i n          >. <Death through Conviction|P u n i s h          >.
+// I tried to fancy up some of the terms.
+// I also changed sickness because of the Sick as ruby, not great to have that repetition.
+// 精神死 gives me some "have no more mind" vibes, rather than "insane".
+// 'Death by sick' sounds about as threatening as 'Death by tummy hurts :((('
+<Death by Immolation|B l a z e         >.
+ <Death by Disease|S i c k            >.
+ <Death by Hemorrhage|B l a d e          >.
+ <Death by Concussion|B r e a k           >.
+ <Death by Dementia|L o s t        >.
+ <Death by Torture|P a i n          >.
+ <Death by Condemnation|P u n i s h          >.
 }
 [sha:ee574b4829e96c569175612ab0cad610df7850a6]{
 -- Page 628, Offset 12507.
 -- 曰く、法が人に割り振った七つの死因。
-As told, the Seven Causes of Death are assigned to man by the law.
+// What law? Lore point?
```

Comment:
no idea and this is not an easy thing to grep for. Ask Petrikow I guess but my first association was the [7 plagues in the bible](https://en.wikipedia.org/wiki/Plagues_of_Egypt) (though notably they don't quite match), so maybe "the laws of god"?